ajax/libs/videojs-ima/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "videojs-ima",
+  "filename": "videojs.ima.min.js",
+  "description": "IMA SDK Plugin for Video.js ",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Google Inc."
+  },
```

Comment:
What I learn before from this: https://github.com/cdnjs/cdnjs/pull/10470#discussion_r98845647
Maybe you can refer to it.
So I still prefer that if using `"author": "Google Inc."` is better.